swagger/v1/swagger.json

```diff
@@ -0,0 +1,14 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "API V1",
+    "version": "v1"
+  },
+  "paths": {
+  },
+  "servers": [
+    {
+      "url": "http://localhost:3000"
+    }
+  ]
+}
```

Comment:
linter is not checking end of file newlines or are they not needed in JSONs?

Reply:
Linter is only for ruby files